### ISSUE TITLE
Please add support pick Month like YearPicker

### ISSUE THREAD

**Issue by borm** [dmtrKovalenko/material-ui-pickers#796]
Duplicate
https://github.com/dmtrKovalenko/material-ui-pickers/issues/355

But maybe you can add some switch case [here](https://github.com/dmtrKovalenko/material-ui-pickers/blob/develop/lib/src/DatePicker/DatePicker.tsx#L147) by some prop

We need something like that
https://material.angular.io/components/datepicker/overview

**Comment by stevez86** [comment]
This is functionality that I need as well. Hoping to allow the user to just pick a month and year (no day)

**Comment by ashanren** [comment]
Up +1

**Comment by shurikdima** [comment]
I agree. i can't use the library without that feature. seems weird not to include such an option when you have the option to only render the calendar.

**Comment by OleksiiKukuruza** [comment]
@dmtrKovalenko do you have plans to add month/year picker?
